.github/workflows/release-native.yml

```diff
@@ -25,9 +25,9 @@ jobs:
         deploy: [
           { os: macOS-13, name: scalafmt-x86_64-apple-darwin.zip },
           { os: macOS-14, name: scalafmt-aarch64-apple-darwin.zip },
-          { os: ubuntu-latest, name: scalafmt-x86_64-pc-linux.zip },
+          { os: ubuntu-22.04, name: scalafmt-x86_64-pc-linux.zip },
```

Comment:
should we do the same in ci.yml? there's a step there which tests that native can be built and that it works.

Reply:
To be on a super safe side we can. Should I do it in this PR? I don't think anything should really break if we don't do that

Reply:
sounds reasonable